example.yaml

```diff
@@ -35,7 +35,7 @@ spec:
           resources:
             limits:
               nvidia.com/gpu: 1 # requesting 1 vGPUs
```

Comment:
`nvidia.com/gpu` Declare how many physical GPUs the pod needs,

`nvidia.com/gpumem`, `nvidia.com/gpucores`  Identifies how much GPU memory and GPU core each physical GPU allocates to the pod